{
    "data_version": "4.0",
    "data_type": "CVE",
    "data_format": "MITRE",
    "CVE_data_meta": {
        "ID": "CVE-2018-10864",
        "ASSIGNER": "secalert@redhat.com",
        "STATE": "PUBLIC"
    },
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "An uncontrolled resource consumption flaw has been discovered in redhat-certification in the way documents are loaded. A remote attacker may provide an existing but invalid XML file which would be opened and never closed, possibly producing a Denial of Service."
            }
        ]
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "Uncontrolled Resource Consumption",
                        "cweId": "CWE-400"
                    }
                ]
            }
        ]
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "vendor_name": "Red Hat",
                    "product": {
                        "product_data": [
                            {
                                "product_name": "Red Hat Certification for Red Hat Enterprise Linux 7",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "0:5.16-20180809.el7",
                                            "version_affected": "!"
                                        },
                                        {
                                            "version_value": "0:5.16-20180809.1.el7",
                                            "version_affected": "!"
                                        }
                                    ]
                                }
                            }
                        ]
                    }
                }
            ]
        }
    },
    "references": {
        "reference_data": [
            {
                "url": "https://access.redhat.com/errata/RHSA-2018:2373",
                "refsource": "MISC",
                "name": "https://access.redhat.com/errata/RHSA-2018:2373"
            },
            {
                "url": "https://access.redhat.com/security/cve/CVE-2018-10864",
                "refsource": "MISC",
                "name": "https://access.redhat.com/security/cve/CVE-2018-10864"
            },
            {
                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1593627",
                "refsource": "MISC",
                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1593627"
            },
            {
                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-10864",
                "refsource": "MISC",
                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-10864"
            }
        ]
    },
    "credits": [
        {
            "lang": "en",
            "value": "This issue was discovered by Riccardo Schirone (Red Hat Product Security)."
        }
    ],
    "impact": {
        "cvss": [
            {
                "attackComplexity": "LOW",
                "attackVector": "NETWORK",
                "availabilityImpact": "LOW",
                "baseScore": 5.3,
                "baseSeverity": "MEDIUM",
                "confidentialityImpact": "NONE",
                "integrityImpact": "NONE",
                "privilegesRequired": "NONE",
                "scope": "UNCHANGED",
                "userInteraction": "NONE",
                "vectorString": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:L",
                "version": "3.0"
            }
        ]
    }
}